yellow_line_follower_controller:
  ros__parameters:
    kp: 0.5263157894736842
    ki: 0.042105263157894736
    kd: 0.16842105263157894
    use_kalman: true
    kalman_q: 0.02
    kalman_r: 0.08
